Yoast/yoast-components
# Merge content analysis in sidebar feature branch into develop

## Summary

This PR can be summarized in the following changelog entry:

* Add `setOverallSeoScore` and `setOverallReadabilityScore` reducer.
* Add help text to the content analysis in the standalone.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Check if the help text is added to the content analysis in the standalone.
* Link this branch to Free. Look in the redux whether the SEO score and readability score are updated correctly.

Requires https://github.com/Yoast/wordpress-seo/pull/10305
